### DISCUSSION TITLE
Option to disable the keyboard navigation of Joyride

### DISCUSSION THREAD

**Discussion by noglitchyo** [gilbarbara/react-joyride#601] [Ideas]
## 🚀 Feature Idea

It would be great to be able to disable the keyboard navigation (described in https://docs.react-joyride.com/accessibility) through a props of Joyride component.

## Motivation

I need to display the tooltip at the same time that I display a form to the user who needs to complete it to go to the next step. 
_The solution to just not display the tooltip is not valid in this scenario since I need to display dynamic information and options in the tooltip as the user progress in the form completion._

- It is usual for a user to use the tabulation key to complete a form.
- It makes less accessible the fact that the tab key can not be use to complete the form

For these reasons, I need to be able to disable the capture of the tab to use it in my form.

**Comment by seth-ament-liftoff** [comment]
I have a similar issue where one stage of the tour asks the user to fill in two inputs located in the spotlight. Since Joyride eats the tab, the user can't tab from the first element to the second.

I've enabled clicking through the spotlight so the user can interact with the app so it would make sense that that would include tabbing to them too. I know the original intention of that is so that the user can click a link but the same tabbing issue applies there.

I wouldn't mind making a PR myself to add spotlight tab support if that'd help. It seems like all that's really required is using the step's target as another element to query in `findValidTabElements` in `scope.js` when `spotlightClicks` is true.

Edit: Actually what I'm looking for might be more appropriate as an issue and not necessarily related to this idea. Let me know what'd be more appropriate cause I don't know what to do.

**Comment by gilbarbara** [comment]
Fixed in v3.0.0

Full migration guide: https://react-joyride.com/docs/migration
What's new: https://react-joyride.com/docs/new-in-v3

Thanks
